TrainingApp/Controller/LanguageViewController.swift

```diff
@@ -128,15 +131,33 @@ final class LanguageViewController: UIViewController, UITableViewDataSource, UIT
             englishUSCheck.isHidden = false
             englishCheck.isHidden = true
             englishUKCheck.isHidden = true
+            Bundle.setLanguage("en")
+            UserDefaults.standard.set("en",forKey: "Language")
+            languageLabel.text = R.string.localizable.language().localizableString("en")
+            suggestedLangLabel.text = R.string.localizable.suggestedLanguages().localizableString("en")
+            englishUSLabel.text = R.string.localizable.englishUS().localizableString("en")
+            polishLabel.text = R.string.localizable.polish().localizableString("en")
+            englishUKLabel.text = R.string.localizable.englishUK().localizableString("en")
+            longTextLabel.text = R.string.localizable.appWillUse().localizableString("en")
+            otherLangLabel.text = R.string.localizable.otherLanguages().localizableString("en")
```

Comment:
i think it is not the best way of using different languages in application (let's get on call and discuss it)

Reply:
@Dagmawit-AHG what about it?

Reply:
I have removed it from the code. The labels can change in language but only if I leave the page and reopen it again.

Reply:
OK👌  Let's leave it to next PR